src/http/services/sync.service.ts

```diff
@@ -109,6 +109,13 @@ export class SyncService {
                         sync.type = DeviceTypes.Lock;
                         sync.traits.push(Traits.LockUnlock);
                         break;
+                    case 'vacuum':
+                        sync.type = DeviceTypes.Vacuum;
+                        sync.traits.push(Traits.StartStop, Traits.Dock);
+                        sync.attributes = {
+                            pausable: true
```

Comment:
shouldn't this param come from the device property? I noticed there is an optional property called pausable.